cclib/attribute_parsers/ccenergies.py

```diff
@@ -34,9 +35,33 @@ def psi4(file_handler, ccdata) -> Optional[dict]:
             return {ccenergies.__name__: np.array(this_ccenergies)}
         return None
 
-        # The geometry convergence targets and values are printed in a table, with the legends
+    @staticmethod
+    def gaussian(file_handler, ccdata) -> Optional[dict]:
+        line = file_handler.last_line
+        if getattr(ccdata, "ccenergies") is None:
+            this_ccenergies = []
+        else:
+            this_ccenergies = []  # note we only save the last ccenergy
+        # Total energies after Coupled Cluster corrections.
+        # Second order MBPT energies (MP2) are also calculated for these runs,
+        # but the output is the same as when parsing for mpenergies.
+        # Read the consecutive correlated energies
+        # but append only the last one to ccenergies.
+        # Only the highest level energy is appended - ex. CCSD(T), not CCSD.
+        parsed_ccenergy = None
+        if line[1:10] == "DE(Corr)=" and line[27:35] == "E(CORR)=":
+            parsed_ccenergy = utils.float(line.split()[3])
+        if line[1:10] == "T5(CCSD)=":
+            line = file_handler.virtual_next()
+            if line[1:9] == "CCSD(T)=":
+                parsed_ccenergy = utils.float(line.split()[1])
+
+        if parsed_ccenergy:
+            this_ccenergies.append(parsed_ccenergy)
+            return {ccenergies.__name__: np.array(this_ccenergies)}
+        return None
 
-    known_codes = ["psi4"]
+    known_codes = ["psi4", "gaussian"]
```

Comment:
could we move this to the top of the class definition for consistency among attributes? This looks like my mistake initially, but just catching it now, my apologies.

Reply:
done!